package.json

```diff
@@ -34,6 +34,19 @@
     "eslint-plugin-prettier": "^4.0.0",
     "lerna": "^4.0.0",
     "prettier": "^2.4.1",
-    "typescript": "^4.5.2"
+    "ts-node": "^10.4.0",
```

Comment:
These dependencies should not be added here, but instead in the JS folder you should create for the nft-packs since they're needed specifically for this project.
We want to structure dependencies in such a manner that they are self contained and we could theoretically move each contract folder like `nft_packs` into a separate repo. Thus as much as possible they should declare what they depend on instead of the root package.json having those deps.

As you see currently those deps are only the ones needed to lint and build.